meta-balena-common/recipes-core/systemd/systemd/65-resin-update-state.rules

```diff
@@ -7,6 +7,16 @@ ENV{ID_FS_LABEL_ENC}=="resin-*|balena-*", IMPORT{program}="/lib/udev/resin_updat
 ENV{ID_FS_LABEL_ENC}=="resin-*|balena-*", ENV{BALENA_FS_USE}=="?*", SYMLINK+="disk/by-state/$env{BALENA_FS_USE}"
 ENV{ID_FS_LABEL_ENC}=="resin-root*|balena-root*", ENV{BALENA_UPDATE_STATE}=="?*", SYMLINK+="disk/by-state/$env{BALENA_UPDATE_STATE}"
 
+IMPORT{cmdline}="flasher"
+ENV{flasher}=="?*", GOTO="balena_uuid_regen_end"
```

Comment:
The above is skipping this for boots with flasher in the cmdline, right?

Reply:
Correct